.classpath

```diff
@@ -12,5 +12,7 @@
 	</classpathentry>
 	<classpathentry kind="var" path="opencv" sourcepath="opencv.sources"/>
 	<classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib-javadoc.jar"/>
```

Comment:
No please
Please cd to the root of your repo and run
```
git checkout master -- .classpath
git add .classpath
git commit
git push
```
The first command gets the version of .classpath master has